### ISSUE TITLE
Fanart (artist slideshow), since the latest update of Kodi, only displays fanart1

### ISSUE THREAD

**Issue by JayDotEll** [manfeed/skin.aeon.tajo#10]
Dear dev,

I just noticed that artist slideshow has stopped playing multiple fanart files. It only shows the first available and stops. This worked before and stopped working after the last (or the two last) Kodi updates for Android.

- Switching to Kodi's default skin resolves and fanart is showing the other fanart files as well without any other change.
- Switching back to aeon tajo brings back the issue of only showing the first available fanart file.

Grateful if you could look into this issue because I strongly want to avoid going back to the default skin.

Regards,
Jay

**Comment by manfeed** [comment]
I just checked and everything is ok on my side (windows). Artist slideshow addon had a big change some time ago and Aeon Tajo was adapted accordingly, so make sure you have the latest version of both, Aeon Tajo and Artist slideshow addon.

**Comment by JayDotEll** [comment]
I had checked that in advance of posting the issue. I have one test device and even reinstalled from scratch with latest version of Kodi, Artist slideshow, scrapers etc.Still with standard skin it works while unfortunately not with Aeon tajo. All settings for the test device done exclusively from Aeon talk. Wondering why it works from default skin...Von meinem Smartphone gesendet.
-------- Ursprüngliche Nachricht --------Von: manfeed <notifications@github.com> Datum: 25.08.20  17:14  (GMT+01:00) An: "manfeed/skin.aeon.tajo" <skin.aeon.tajo@noreply.github.com> Cc: JayDotEll <styl@ish.de>, Author <author@noreply.github.com> Betreff: Re: [manfeed/skin.aeon.tajo] Fanart (artist slideshow), since the latest update of Kodi, only displays fanart1 (#10) 
I just checked and everything is ok on my side (windows). Artist slideshow addon had a big change some time ago and Aeon Tajo was adapted accordingly, so make sure you have the latest version of both, Aeon Tajo and Artist slideshow addon.
